pkg/didcomm/messaging/service/http/models.go

```diff
@@ -0,0 +1,22 @@
+/*
+ *
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * /
+ *
+ */
+
+package http
+
+// httpOverDIDCommMsg is incoming DIDComm message for http-over-didcomm message types
+// Reference:
+//  https://github.com/hyperledger/aries-rfcs/blob/master/features/0335-http-over-didcomm/README.md#message-format
+type httpOverDIDCommMsg struct {
+	ID          string            `json:"@id"`
+	Method      string            `json:"method"`
+	ResourceURI string            `json:"resource-uri,omitempty"`
+	Version     string            `json:"version"`
+	Headers     map[string]string `json:"headers"`
```

Comment:
`headers` is defined as an array in the RFC. @Moopli should update the RFC and specify that it's an array of some object composed of key-value pairs.

Reply:
I already posted on comment on his RFC and notified him.

Reply:
And also, changed map to array of `name-value` since headers can be duplicate.

Reply:
I have a PR open for this https://github.com/hyperledger/aries-rfcs/pull/399